owl/EASE-OBJ.owl

```diff
@@ -455,6 +666,54 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/EASE-OBJ.owl#Barrier -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-OBJ.owl#Barrier">
+        <rdfs:subClassOf rdf:resource="http://www.ease-crc.org/ont/EASE-OBJ.owl#Excluder"/>
+        <rdfs:comment>A tool used as a barrier to prevent access to a protected space, or to prevent enclosed objects from escaping.</rdfs:comment>
```

Comment:
I see you either thought along similar lines to me or adapted some of the disposition branch I made into these commits :)

The reason I invented the Barrier concept though, as opposed to Robert's Inclusion/Exclusion, was because I didn't see that Containment and other similar schemas are about Exclusion or Inclusion.

If I want to prevent access to a protected space, I'm definitely excluding the stuff that's out. But if I want to prevent enclosed objects escaping, I'm not being exclusionary.

Reply:
I thought along similar lines :)

Reply:
Well you could argue that you exclude the enclosed objects from the rest of the world, right? I was rather seeing *Inclusion* as "make an object part of a bigger whole" -- e.g. storing it together with other objects in a common compartment. And *Exclusion* as either prevent from entering, leaving or seeing some space.